integration/headers-test.ts

```diff
@@ -256,7 +245,6 @@ test.describe("headers export", () => {
     expect(JSON.stringify(Array.from(response.headers.entries()))).toBe(
       JSON.stringify([
         ["content-type", "text/html"],
-        ["x-child-loader", "success"],
```

Comment:
No more headers from children in these tests since we're using the parent `headers` function